tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestDataTest.java

```diff
@@ -0,0 +1,236 @@
+                            namePattern.append(parts.get(1)).append('=').append(parts.get(2));
+                            break;
+                        case "expectedResult":
+                            expectedResult = parts.get(1); // TODO unescape ";"
+                            if (pnState == PersonNameState.haveName) {
+                                sampleName =
+                                        SimpleNameObject.from(
+                                                NAME_PATTERN_SEPERATOR, namePattern.toString());
+                                pnState = PersonNameState.haveResult;
+                            }
+                            break;
+                        case "parameters":
+                            // order=sorting; length=long; usage=referring; formality=formal
+                            StringBuilder parameters = new StringBuilder();
+                            for (int i = 1; i < parts.size(); ++i) {
+                                String value = parts.get(i);
+                                if (value.equals("none")) { // special order indicating use locale)
+                                    value =
+                                            pnf.getOrderFromLocale(sampleName.getNameLocale())
+                                                    .toString();
+                                }
+                                String key = valueToKey.get(value);
+                                if (key == null) {
+                                    int debug = 0;
+                                }
+                                parameters.append(key).append('=').append(value).append("; ");
+                            }
+                            String actual;
+                            try {
+                                FormatParameters formatParameters =
+                                        FormatParameters.from(parameters.toString());
+
+                                actual = pnf.format(sampleName, formatParameters).replace("ᵛ", "");
+                            } catch (Exception e) {
+                                actual = e.getMessage();
+                            }
+                            if (!assertEquals(
+                                    localeId + " (" + count + ") ", expectedResult, actual)) {
+                                int debug = 0;
+                            }
+                            break;
+                        case "endName":
+                            pnState = PersonNameState.start;
+                            break;
+                        default:
+                            throw new IllegalArgumentException("Unknown value");
+                    }
+                }
+            } catch (Exception e) {
+                throw new IllegalArgumentException(localeId + " (" + count + ") " + line, e);
+            }
+        }
+    }
+
+    /** Make sure this is the last test */
+    public void testΩ() {
```

Comment:
A Greek letter in a function name!  Wow!  I keep forgetting you can do things like that in Java...

Reply:
And now many other programming languages: Swift, and even C++